<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditOperation</class>
 <widget class="QDialog" name="EditOperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="from_cod">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="MyDateEdit" name="dateEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>To account:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="AccountComboBox" name="to_Account"/>
     </item>
     <item row="1" column="1">
      <widget class="AgentComboBox" name="agent_comboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="currencyEdit" name="summSpinBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="AccountComboBox" name="from_account"/>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="descrEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Agent:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From account:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Summ:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="warning">
       <property name="text">
        <string>Nedostatochno sredstv na schete</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="to_cod">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Kurs:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="from_to">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="currencyEdit" name="kursEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="OkPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CancelPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OkPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/okpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AccountComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/accountcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyDateEdit</class>
   <extends>QDateEdit</extends>
   <header location="global">widgets/mydateedit.h</header>
  </customwidget>
  <customwidget>
   <class>currencyEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/currencyedit.h</header>
  </customwidget>
  <customwidget>
   <class>CancelPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/cancelpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AgentComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/agentcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>from_account</tabstop>
  <tabstop>agent_comboBox</tabstop>
  <tabstop>to_Account</tabstop>
  <tabstop>summSpinBox</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>descrEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
